### PR TITLE
left join complete

### PR DESCRIPTION
**Pull request by paronicholas**
- [x] Top-level README “Table of Contents” is updated
- [x] Feature tasks for this challenge are completed
- [x] Unit tests written and passing
    - [x] “Happy Path” - Expected outcome
    - [x] Expected failure
    - [x] Edge Case (if applicable/obvious)
- [x] README for this challenge is complete
    - [x] Summary, Description, Approach & Efficiency, Solution, Resources
    - [x] Link to code
    - [x] Link to tests

### CODE REVIEW

File: code401challenges/src/main/java/hashTable/LeftJoin.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package hashTable;
+
+public class LeftJoin {
+  public static String[][] leftJoin(HashTable<String,String> hashTable1, HashTable<String,String> hashTable2) {
+    Node<String,String>[] map = hashTable1.getMap();
```
Comment (evanslaton) [inline code401challenges/src/main/java/hashTable/LeftJoin.java]: `.getMap()` is neat! You might consider making this a protected method. You wouldn't want a user to be able to access and modify the hashmap's array directly.
